src/libs/searchCountryOptions.js

```diff
@@ -8,15 +8,15 @@ import CONST from '../CONST';
  * @returns {Object[]} An array of countries/states sorted based on the search query
  */
 function searchCountryOptions(searchValue, countriesData) {
-    const trimmedSearchValue = searchValue.toLowerCase().replaceAll(CONST.REGEX.NON_ALPHABETIC_AND_NON_LATIN_CHARS, '');
-    if (trimmedSearchValue.length === 0) {
+    const trimmedSearchValue = StringUtils.sanitizeString(searchValue);
+    if (_.isEmpty(trimmedSearchValue)) {
         return [];
     }
 
-    const filteredData = _.filter(countriesData, (country) => country.searchValue.includes(trimmedSearchValue));
+    const filteredData = _.filter(countriesData, (country) => _.includes(StringUtils.sanitizeString(country.searchValue), trimmedSearchValue));
```

Comment:
```suggestion
    const filteredData = _.filter(countriesData, (country) => _.includes(country.searchValue, trimmedSearchValue));
```

No need to sanitize again, right?